### DISCUSSION TITLE
Change separation space between choices and textbox

### DISCUSSION THREAD

**Discussion by mak448a** [dialogic-godot/dialogic#2270] [Q&A] [answered]
How do I change the separation space between choices and the textbox? They're touching.

**Comment by Jowan-Spooner** [comment, marked as answer]
These things are defined by the styles. You might have to customize the choice layer. 
See the [Styles and Layers documentation](https://docs.dialogic.pro/styles-and-layouts.html).

**Reply by mak448a** [reply]
I wasn't able to find it there.
![image](https://github.com/dialogic-godot/dialogic/assets/94062293/cd267885-8475-41c8-924f-5af1ab10ecfb)

**Reply by mak448a** [reply]
For whoever comes back here, they added this feature here.
